site/content/docs/standard/designblocks/admin/index.html

```diff
@@ -21,7 +21,7 @@
     <div class="mb-12">
       <div
         class="relative overflow-hidden bg-no-repeat bg-cover ripple shadow-lg rounded-lg mb-6 flex items-center"
-        data-mdb-ripple-color="true" data-mdb-ripple-color="light"
+        data-te-ripple-init
```

Comment:
Is it an intentional omission of the `data-te-ripple-color="light"` attribute in design blocks?

Reply:
Yes, it is. I decided to add only `data-te-ripple-init` because images are bright and the ripple effect in light colour was invisible.